zetaclient/tx.go

```diff
@@ -51,6 +51,8 @@ func (b *ZetaCoreBridge) WrapMessageWithAuthz(msg sdk.Msg) (sdk.Msg, AuthZSigner
 }
 
 func (b *ZetaCoreBridge) PostGasPrice(chain common.Chain, gasPrice uint64, supply string, blockNum uint64) (string, error) {
+	// double the gas price to avoid gas price spike
+	gasPrice = gasPrice * 2
```

Comment:
As a long term solution, is it something we could consider as part of the chain params so we can update this value without upgrade.

`GasPriceMultiplier`
Used for chains where the gas price must be unstable or where the gas price increase mechanism can't work.

cc @brewmaster012

Reply:
@lumtis maybe we can track it here: https://github.com/zeta-chain/node/issues/1700
This cherry-pick PR is purely to sync `develop` and `mainnet`